Cartfile

```diff
@@ -1,3 +0,0 @@
-github "ReactiveX/RxSwift" ~> 6.0
-github "Quick/Quick"
-github "Quick/Nimble"
```

Comment:
Doesn't this remove Carthage support for this library altogether?

Reply:
I have nearly zero experience with using Carthage, except to remove it from preexisting projects 😅 

If you believe we should maintain support for it, I would value any insight you could provide if you're familiar with it. Otherwise I can dive into figuring that out